modules/serverless-configuring-progress-deadline-serving.adoc

```diff
@@ -0,0 +1,40 @@
+// Module included in the following assemblies:
+//
+// * serverless/knative-serving/config-applications/startup-probes-for-serving.adoc
+
+:_content-type: PROCEDURE
+[id="serverless-configuring-progress-deadline-serving_{context}"]
+= Configuring progress deadline
+
+You can configure progress deadline settings to specify the maximum time allowed for your deployment to progress before the system reports a failure for the Knative Revision. This time limit can be specified in seconds or minutes.
+
+To configure the progress deadline effectively, consider the following parameters:
+
+ * `initialDelaySeconds`
+ * `failureThreshold` 
+ * `periodSeconds`
+ * `timeoutSeconds`
+
+If the initial scale is not achieved within the specified time limit, the Knative Autoscaler component scales the revision to `0`, and the Knative service enters a terminal `Failed` state.
+
+By default, the progress deadline is set to 600 seconds. This value is specified as a Golang `time.Duration` string and must be rounded to the nearest second.
+
+.Procedure
+
+* To configure the progress deadline setting, use an annotation in your deployment configuration.
++
+.Example of progress deadline set to 60 seconds
+[source,yaml]
+----
+apiVersion: serving.knative.dev/v1
+kind: Service
+...
+spec:
+  template:
+    metadata:
+       annotations:
+            serving.knative.dev/progress-deadline: "60s" 
```

Comment:
You mention that it can be done in seconds or minutes. Would it be worthwhile to give that info in a callout below.

Like I'm assuming that if they wanted to specify it in minutes, it would be something like `10m`. But that would be good to explicitly note here.